controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1307,6 +1321,49 @@ int cli_bml::get_dcs_scan_results_caller(int numOfArgs)
     return -1;
 }
 
+/**
+ * Caller function for client_get_client_list_caller.
+ *
+ * @param [in] numOfArgs Number of received arguments
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client_list_caller(int numOfArgs)
+{
+    if (numOfArgs == 0) {
+        return client_get_client_list();
+    }
+    return -1;
+}
+
+/**
+ * Caller function for client_set_client_caller.
+ *
+ * @param [in] numOfArgs Number of received arguments
+ * @return 0 on success.
+ */
+int cli_bml::client_set_client_caller(int numOfArgs)
+{
+    if (numOfArgs == 4) {
+        return client_set_client(args.stringArgs[0], args.intArgs[1], args.intArgs[2] == 1,
```

Comment:
originally I liked how you immediately changed to bool where you could!

but since these can be optional (as commented above) this implementation of `client_set_client` should change to support optional-cli-function-params.

Reply:
Changed to support the optional parameters (as not-configured)